plugin/access.lua

```diff
@@ -85,13 +82,13 @@ end
 function _M.run(config)
 
     -- If there is already an authorization header, eg for mobile clients, return immediately
-    local auth_header = ngx.req.get_headers()['Authorization']
+    local auth_header = ngx.req.get_headers()['AUthorization']
```

Comment:
typo? `AU`

By the way - it looks like this could be changed to `ngx.var.http_authorization` according to the docs: https://github.com/openresty/lua-nginx-module#ngxreqget_headers It might be better performance-wise since `get_headers` maps all headers to a lua table.